### PR TITLE
Gethyperlinkfrompath

### PR DESCRIPTION
**Pull request by sahandfarahani**

### CODE REVIEW

File: src/utils/getHyperLinkFromPath.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/** getHyperLinkFromPath generates the clickable link to a specific folder
```
Comment (ralph-dev) [inline src/utils/getHyperLinkFromPath.ts:1]:
```suggestion
/** generates the clickable link to a specific folder
```

File: src/utils/getHyperLinkFromPath.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ * @param {string} path  The path of a specific folder like "src/components/reusable"
```
Comment (ralph-dev) [inline src/utils/getHyperLinkFromPath.ts:2]:
```suggestion
 * @param {string} path - The path of a specific folder like "src/components/reusable"
```

File: src/utils/test.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { expect } from "chai";
+import getHyperLinkFromPath from "./getHyperLinkFromPath";
+
+describe("getHyperLinkFromPath Function", () => {
```
Comment (ralph-dev) [inline src/utils/test.ts:4]: You need more test cases to handle EDGE cases

EX:
src
./src
./src/
./src////
/.wwqe
.//src/
Always good to capture majority of test cases

Comment (sahandfarahani) [reply src/utils/test.ts:4]: so 
./src
./src/
./src////
/.wwqe
.//src/
obviously are wrong because github API wouldn't generate them... but I haven't predict anything inside the function to check if the input is correct or throws an error... should I provide such thing?

Comment (ralph-dev) [reply src/utils/test.ts:4]: Yes your function should always handle edge cases such as bad paths that aren't supported (Throw an Error)

File: src/utils/getHyperLinkFromPath.ts [hunk 4]
```diff
@@ -1,15 +1,24 @@
 /** generates the clickable link to a specific folder
  * @param {string} path - The path of a specific folder like "src/components/reusable"
+<<<<<<< HEAD
```
Comment (ralph-dev) [inline src/utils/getHyperLinkFromPath.ts:3]: Fix Merge Conflicts

File: src/utils/test.ts [hunk 5]
```diff
@@ -2,8 +2,53 @@ import { expect } from "chai";
...
+  it("should return Error", () => {
+    expect(() => getHyperLinkFromPath("./src")).to.throw();
+  });
+});
+describe("getHyperLinkFromPath Function", () => {
+  it("should return Error", () => {
+    expect(() => getHyperLinkFromPath("./src/")).to.throw();
+  });
```
Comment (ralph-dev) [inline src/utils/test.ts:26]: @sahandfarahani @Manpreet-Bhatti This is not how we structure tests with the same name describes / multiple describes?

File: src/utils/test.ts [hunk 6]
```diff
@@ -6,48 +6,28 @@ describe("getHyperLinkFromPath Function", () => {
...
-describe("getHyperLinkFromPath Function", () => {
   it("should return Error", () => {
     expect(() => getHyperLinkFromPath(".//src/")).to.throw();
   });
-});
-
-describe("getHyperLinkFromPath Function", () => {
   it("should return Error", () => {
     expect(() => getHyperLinkFromPath("src//c")).to.throw();
   });
```
Comment (ralph-dev) [inline src/utils/test.ts]: Try adding more descriptive titles to the tests. Edge Case of Extra Slash, Edge Case of Double Slash etc
